app/Domain/Source/Events/UpdateSourceEvent.php

```diff
@@ -15,21 +15,25 @@ class UpdateSourceEvent extends DataTransferObject implements ShouldBeStored
     /** @var string */
     public $url;
 
-    public function __construct(string $source_uuid, string $url)
+    /** @var string|null */
+    public $language;
+
+    public function __construct(string $source_uuid, string $url, ?string $language)
```

Comment:
I'm not sure but I think this question mark means it's nullable and optional. I will check that.

Reply:
`?` only means you're allowed to pass `null` instead of the type, while `= null` means the parameter can be omitted, and its default value will be `null`.

There's a quirk in PHP where `string $foo = null` also allows you to pass in `null` explicitly. That's something that might change with PHP 8, so I prefer `?string $foo = null`